tests/test_helpers.py

```diff
@@ -77,9 +64,9 @@ def __init__(
         # let the dataclasses code initialize `data`.
         self.__default_init__(  # pylint: disable=E1101
             *args,
-            provenance=_to_list(provenance),
-            source_url=_to_list(source_url),
-            source=_to_list(source),
+            provenance=to_list(provenance),
```

Comment:
nit: i think importing this as `combined_datasets` and calling `combined_datasets.to_list` here is a bit more clear when reading through but low priority

Reply:
Done in 7a6c56e0